test_scripts/API/Transfer_RPC_with_invalid_image/013_GetWayPoints.lua

```diff
@@ -0,0 +1,91 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0042-transfer-invalid-image-rpc.md
+--
+-- Requirement summary:TBD
+--
+-- Description:
+-- In case:
+-- 1. Mobile app requests GetWayPoints with image that is absent on file system
+-- SDL must:
+-- 1. transfer this RPC to HMI for processing
+-- 2. transfer the received from HMI response (WARNINGS, message: “Requested image(s) not found”) to mobile app
```

Comment:
I can't imagine a reason that the HMI would respond with WARNINGS in this case, since the app has no control over the content of the waypoints. No warning should be issued because the app has no control over this matter.

Reply:
In `Navigation.GetWayPoints` response HMI sends path to image, so for example if there is no image in defined folder, so HMI can send this path by also send WARNING result code.

Reply:
Yes, but why send a warning if the app can do nothing about it? To be entirely honest, there isn't much of a reason to send a dynamic image in the `GetWayPoints` response, since the app is unable to access this image.

Reply:
@jacobkeeler , updated in 177f1414e291055e51b8a2b205d988766aa27bcf, checks that SDL only transfer response to mobile app without image validation.